src/pydocstyle/config.py

```diff
@@ -303,8 +303,12 @@ def _read_configuration_file(self, path):
         should_inherit = True
 
         if parser.read(path) and self._get_section_name(parser):
+            all_options = [o for o in self._parser.option_list]
```

Comment:
Can't this be just `all_options = self_parser.option_list[:]`?

Reply:
But mom, all the cool kids are doing it!